asdf/tests/test_fits_embed.py

```diff
@@ -463,6 +463,29 @@ def test_array_view_compatible_dtype(tmp_path):
             af.write_to(file_path)
 
 
+def test_hdu_link_independence(tmp_path):
```

Comment:
Could you add a nice description/documentation string like you did for the `test_resave_breaks_hdulist_tree_array_link` test? This isn't necessary, but the one in the other test was very helpful for me.

Reply:
Great suggestion! I added a docstring to the test. Let me know if it looks like what you had in mind.

https://github.com/braingram/asdf/blob/bug/hdu_array_link_breaking/asdf/tests/test_fits_embed.py#L467-L476